espRFID/UpdateACLTask.cpp

```diff
@@ -96,6 +106,12 @@ void UpdateACLTask::manualUpdate()
     setRunTime(nextAutomatedTime);
     latchTask.blinkDoor();
   }
+  else
+  {
+    incRunTime(nextRetryInterval); 
+    nextAutomatedTime = runTime;
```

Comment:
You don't want this line.  This will make it do an "automatic update" in 10 seconds.  The main difference is that in the automatic update retry it won't blink the door which will be confusing to someone doing an update.